apps/src/sites/studio/pages/init/loadJavalab.js

```diff
@@ -2,11 +2,15 @@ import appMain from '@cdo/apps/appMain';
 import {singleton as studioApp} from '@cdo/apps/StudioApp';
 import Javalab from '@cdo/apps/javalab/Javalab';
 import levels from '@cdo/apps/javalab/levels';
+import skins from '@cdo/apps/maze/skins';
 
 export default function loadJavalab(options) {
   options.isEditorless = true;
   const javalab = new Javalab();
 
   javalab.injectStudioApp(studioApp());
+  // TODO: Update this to "neighborhood" once the skin has been created.
+  options.skinsModule = skins;
+  options.skinId = "birds";
```

Comment:
it doesn't look like it, but are these skinIds captured in a constant/enum anywhere? i see them hard-coded throughout our frontend 😢

Reply:
Hmm. Not that I know of. I'm going to leave this as out-of-scope for now since this isn't the value we'll be using anyway.

Reply:
yeah, definitely out-of-scope. i added [STAR-1565](https://codedotorg.atlassian.net/browse/STAR-1565) to clean this up